tests/puppeteer/test/utils/helpers.js

```diff
@@ -0,0 +1,31 @@
+require('./globals');
+const chai = require('chai');
+chai.use(require('chai-string'));
+global.expect = chai.expect;
+
+const puppeteer = require('puppeteer');
+
+global.test = (name, instructions) => it(name, () => {
+  return instructions().catch();
+});
+
+global.scenario = (name, tests, close = false) =>
+  describe(name, async () => {
+    before(async () => {
```

Comment:
Maybe add an await to before function

Reply:
Je viens de tester, mocha n'aime pas ça du tout et me met `0 passing` donc on oublie :-D

Reply:
OK